nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -109,7 +109,7 @@
 
 - `pass` now does not contain `password-store.el`.  Users should get `password-store.el` from Emacs lisp package set `emacs.pkgs.password-store`.
 
-- `services.knot` now supports `.settings` from RFC42.  The change is not 100% compatible with the previous `.extraConfig`.
+- `services.knot` now supports `.settings` from RFC42.  The previous `.extraConfig` still works the same, but it throws a warning now.
```

Comment:
```suggestion
- `services.knot` now supports `.settings` from RFC42. The previous `.extraConfig` still works the same, but it displays a warning now.
```
we should avoid throw here as it could confuse people.

Reply:
OK, we can change this line, but I'm still unsure whether that warning should be changed or even removed.